api/v1alpha1/bundledeployment_types.go

```diff
@@ -86,6 +86,9 @@ type BundleDeploymentStatus struct {
 //+kubebuilder:printcolumn:name=Provisioner,type=string,JSONPath=`.spec.provisionerClassName`,priority=1
 
 // BundleDeployment is the Schema for the bundledeployments API
+// TODO(varsha): Marking this as unserved version for now. Remove this definition during cleanup, when the all the
+// work on alpha2 is done.
+// +kubebuilder:unservedversion
```

Comment:
This PR does not wire in the reconciler to the controller yet. Can make this as a storage version until that is done.